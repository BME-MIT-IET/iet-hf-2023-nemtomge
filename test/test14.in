loadmap map.json 
start V1 
set Scientist V1 PL1 
set Gear PL1 Coat 
touch V1 
1 
show Gear V1
exit
