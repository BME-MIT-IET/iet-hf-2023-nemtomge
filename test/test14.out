Map from file map.json loaded
The game has been started. Players are: V1 
Scientist V1 set on field PL1 
Gear Coat has been placed on field PL1 
Choose one to pick: 
0 - cancel
1 - Coat
V1's gears are:
Coat
